### ISSUE TITLE
CREATE TABLE query does not return any success response

### ISSUE THREAD

**Issue by gaurav274** [georgia-tech-db/evadb#1186]
### Search before asking

- [X] I have searched the EvaDB [issues](https://github.com/georgia-tech-db/eva/issues) and found no similar bug report.

### Bug

```
 create_table_query = """
           CREATE TABLE IF NOT EXISTS HomeRentals (
               number_of_rooms INTEGER,
               number_of_bathrooms INTEGER,
               sqft INTEGER,
               location TEXT(128),
               days_on_market INTEGER,
               initial_price INTEGER,
               neighborhood TEXT(128),
               rental_price FLOAT(64,64)
           );"""

cur.query(create_table_query).df()

Empty DataFrame
Columns: []
Index: []
```

### Environment

- EvaDB - 0.3.6

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by xzdandy** [comment]
How does the following sounds?
If created:
 - `Table HomeRentals has been created.` 
If already exists
- `Table HomeRentals is already existed. Nothing added.`

**Comment by uname-s** [comment]
I would rephrase:
`Table HomeRentals is already existed. Nothing added.`
as:
`Table HomeRentals already exists.`

The "Nothing added." part feels redundant but I can see the logic in being explicit that no action was taken.
